<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_prod</class>
 <widget class="QDialog" name="add_prod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Введите название товара:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Name_prod"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Введите артикул товара:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Art_prod">
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Введите цену(в рублях):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="price_prod"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Введите максимальное количество товаров в магазине:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Count_prod">
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_prod_button">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Добавить продукт</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
